inc/ui/namespace.php

```diff
@@ -109,7 +109,7 @@ function render_page() {
 							class="<?php echo $group === $current_group ? 'current' : '' ?> <?php echo $group === $current_group && ! $current_page_id ? 'active' : '' ?>"
 						>
 							<a
-								href="<?php echo add_query_arg( [ 'group' => $group, 'id' => '' ] ) ?>"
+								href="<?php echo add_query_arg( [ 'group' => $group, 'id' => '' ] ) // @codingStandardsIgnoreLine ?>"
```

Comment:
Could we `esc_url` instead of ignoring?

Reply:
The error is the assoc array with 2 items in it